processing/src/main/java/io/druid/query/timeboundary/TimeBoundaryQuery.java

```diff
@@ -34,21 +35,16 @@
 import io.druid.query.spec.MultipleIntervalSegmentSpec;
 import io.druid.query.spec.QuerySegmentSpec;
 import org.joda.time.DateTime;
-import org.joda.time.Interval;
 
 import java.nio.ByteBuffer;
-import java.util.Collections;
 import java.util.List;
 import java.util.Map;
 
 /**
  */
 public class TimeBoundaryQuery extends BaseQuery<Result<TimeBoundaryResultValue>>
 {
-  public static final Interval MY_Y2K_INTERVAL = new Interval(
-      new DateTime("0000-01-01"),
-      new DateTime("3000-01-01")
-  );
+  private static final QuerySegmentSpec DEFAULT_SEGMENT_SPEC = new MultipleIntervalSegmentSpec(Intervals.ONLY_ETERNITY);
```

Comment:
I don't want to expose it, because this spec is harmful if used for topN, groupBy, timeseries queries. So I limit it's scope to "special" query types.